### DISCUSSION TITLE
这绳网居然还能用？

### DISCUSSION THREAD

**Discussion by FFARD** [share121/inter-knot#5760] [公告]
怎么大家还没走光呢？

**Comment by rnfmabj-rnfmabj** [comment]
绳网是游戏里的灰色地带，而绳网又因为法律问题要被封，而绳网却说封没封，在灰色地带，令人忍俊不禁

**Comment by 1155993355** [comment]
不知道是不是我才疏学浅，我没搞清楚怎么发帖

**Reply by FFARD** [reply]
![Screenshot_20241215_214432](https://github.com/user-attachments/assets/ed7ff9b0-c378-43e8-b672-e642eaa4fcfc)

**Comment by 1155993355** [comment]
Thank you

**Comment by Fulinze** [comment]
这个投票是必须填吗

**Comment by he222111** [comment]
绳网还有人吗感觉都跑了呃
